modules/cookbook/pages/recipe-configmgmt.adoc

```diff
@@ -0,0 +1,113 @@
+[[cookbook-config-mgmt]]
+= Configuration Management
+
+You can use configuration files and channels to manage configuration for your clients, rather than configuring each client manually.
+This recipe shows you how to use the {productname} {webui} to create a centrally-managed configuration file, assign it to a Salt client, and apply the configuration.
+
+
+
+== Problem
+
+If you are managing a lot of clients, you probably don't want to manually apply configuration settings to each of them in turn.
+Configuration channels are used to organize configuration files.
+You can subscribe clients to configuration channels, and deploy configuration files as required.
+
+
+
+== Solution
+
+When you have completed this recipe, you will have a configuration channel containing a configuration file, have assigned clients to the channel, and applied the configuration successfully.
+
+
+
+== Before you begin
+
+Before you start, you should already have:
+
+* A {productname} Server installed, that you can access using the {webui}.
+* At least one Salt client registered to your server.
+* Appropriate subscriptions from http://scc.suse.com for the products you are using.
+
+This recipe uses a centrally managed configuration file and a Salt state.
+You can also use locally-managed configuration files,  traditional clients, and different methods, to get more flexibility in how you manage configuration in your environment.
+For more information about the different ways to manage configuration, see xref:client-configuration:configuration-management.adoc[].
+
+
+
+== Steps
+
+Create a new configuration channel and file::
+
+. In the {productname} {webui}, navigate to menu:Configuration[Channels] and click btn:[Create State Channel].
+. Type a name, label, and description for your configuration file, and type the contents of your configuration file.
```

Comment:
I think you first want to enter the details of the channel, the cfg file will come later.

Reply:
Not really, it's a very weird UI.